2024-05-18 15:56:11,991 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2024-05-18 15:56:11,991 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Configure stats pid to 18296
2024-05-18 15:56:11,991 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Loading settings from C:\Users\ernst\.config\wandb\settings
2024-05-18 15:56:11,991 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Loading settings from c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\settings
2024-05-18 15:56:11,991 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-18 15:56:11,991 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-18 15:56:11,992 INFO    MainThread:18296 [wandb_init.py:_log_setup():524] Logging user logs to c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155611-j7r3iz8i\logs\debug.log
2024-05-18 15:56:11,992 INFO    MainThread:18296 [wandb_init.py:_log_setup():525] Logging internal logs to c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155611-j7r3iz8i\logs\debug-internal.log
2024-05-18 15:56:11,992 INFO    MainThread:18296 [wandb_init.py:init():564] calling init triggers
2024-05-18 15:56:11,992 INFO    MainThread:18296 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-05-18 15:56:11,992 INFO    MainThread:18296 [wandb_init.py:init():589] re-initializing run, found existing run on stack: a380w12h
2024-05-18 15:56:11,993 INFO    MainThread:18296 [wandb_run.py:_finish():1962] finishing run er-wnb-1a33y/table-test/a380w12h
2024-05-18 15:56:11,993 INFO    MainThread:18296 [jupyter.py:save_history():445] not saving jupyter history
2024-05-18 15:56:11,993 INFO    MainThread:18296 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-18 15:56:11,993 INFO    MainThread:18296 [wandb_init.py:_jupyter_teardown():452] cleaning up jupyter logic
2024-05-18 15:56:11,993 INFO    MainThread:18296 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-05-18 15:56:11,993 INFO    MainThread:18296 [wandb_run.py:_restore():2185] restore
2024-05-18 15:56:11,993 INFO    MainThread:18296 [wandb_run.py:_restore():2191] restore done
2024-05-18 15:56:15,705 INFO    MainThread:18296 [wandb_run.py:_footer_history_summary_info():3760] rendering history
2024-05-18 15:56:15,705 INFO    MainThread:18296 [wandb_run.py:_footer_history_summary_info():3792] rendering summary
2024-05-18 15:56:15,707 INFO    MainThread:18296 [wandb_run.py:_footer_sync_info():3719] logging synced files
2024-05-18 15:56:15,728 INFO    MainThread:18296 [wandb_init.py:init():614] starting backend
2024-05-18 15:56:15,728 INFO    MainThread:18296 [wandb_init.py:init():618] setting up manager
2024-05-18 15:56:15,731 INFO    MainThread:18296 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-05-18 15:56:15,731 INFO    MainThread:18296 [wandb_init.py:init():624] backend started and connected
2024-05-18 15:56:15,739 INFO    MainThread:18296 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-05-18 15:56:15,739 INFO    MainThread:18296 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-18 15:56:15,739 INFO    MainThread:18296 [wandb_init.py:init():716] updated telemetry
2024-05-18 15:56:15,758 INFO    MainThread:18296 [wandb_init.py:init():749] communicating run to backend with 90.0 second timeout
2024-05-18 15:56:17,126 INFO    MainThread:18296 [wandb_run.py:_on_init():2254] communicating current version
2024-05-18 15:56:17,197 INFO    MainThread:18296 [wandb_run.py:_on_init():2263] got version response upgrade_message: "wandb version 0.17.0 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2024-05-18 15:56:17,197 INFO    MainThread:18296 [wandb_init.py:init():800] starting run threads in backend
2024-05-18 15:56:17,273 INFO    MainThread:18296 [wandb_run.py:_console_start():2233] atexit reg
2024-05-18 15:56:17,274 INFO    MainThread:18296 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-05-18 15:56:17,274 INFO    MainThread:18296 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-05-18 15:56:17,274 INFO    MainThread:18296 [wandb_run.py:_redirect():2178] Redirects installed.
2024-05-18 15:56:17,274 INFO    MainThread:18296 [wandb_init.py:init():841] run started, returning control to user process
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Current SDK version is 0.16.0
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Configure stats pid to 18296
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Loading settings from C:\Users\ernst\.config\wandb\settings
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Loading settings from c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\settings
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_init.py:_log_setup():524] Logging user logs to c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\logs\debug.log
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_init.py:_log_setup():525] Logging internal logs to c:\Users\ernst\Documents\02-Work\01-Projects\MANTRA\wandb\run-20240518_155754-b1l3fxxt\logs\debug-internal.log
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001A998135F30>
2024-05-18 15:57:54,437 INFO    MainThread:18296 [wandb_init.py:init():564] calling init triggers
2024-05-18 15:57:54,438 INFO    MainThread:18296 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2024-05-18 15:57:54,438 INFO    MainThread:18296 [wandb_init.py:init():589] re-initializing run, found existing run on stack: j7r3iz8i
2024-05-18 15:57:54,438 INFO    MainThread:18296 [wandb_run.py:_finish():1962] finishing run er-wnb-1a33y/table-test/j7r3iz8i
2024-05-18 15:57:54,438 INFO    MainThread:18296 [wandb_run.py:_atexit_cleanup():2202] got exitcode: 0
2024-05-18 15:57:54,439 INFO    MainThread:18296 [wandb_run.py:_restore():2185] restore
2024-05-18 15:57:54,439 INFO    MainThread:18296 [wandb_run.py:_restore():2191] restore done
2024-05-18 15:57:57,462 INFO    MainThread:18296 [wandb_run.py:_footer_history_summary_info():3760] rendering history
2024-05-18 15:57:57,462 INFO    MainThread:18296 [wandb_run.py:_footer_history_summary_info():3792] rendering summary
2024-05-18 15:57:57,464 INFO    MainThread:18296 [wandb_run.py:_footer_sync_info():3719] logging synced files
